local-macros/src/lib.rs

```diff
@@ -91,9 +130,23 @@ pub fn ast_type(_attr: TokenStream, item: TokenStream) -> TokenStream {
             let variant_names_10 = variant_names.clone();
             let variant_names_11 = variant_names.clone();
             let variant_names_12 = variant_names.clone();
+            let variant_names_13 = variant_names.clone();
+            let variant_names_14 = variant_names.clone();
```

Comment:
If you `collect` `variant_names` into a `Vec`, you can reuse it and the clones aren't necessary.

Reply:
Per a later PR I ended up getting rid of these by using a reference per an earlier comment of yours

Ah actually I think you're pointing at the real reason it started working (because I believe I updated it to be a reference to a `Vec`), if it was an iterator before then it would've gotten consumed by a single usage but a `Vec` can be reused?

Reply:
Yeah, I think that's it.